src/core/balance.c

```diff
@@ -6,6 +6,8 @@
 #include "core/balance.h"
 #include "libbase58.h"
 
+// Note: colored coins are not yet included to Chrysalis, only Pollen
```

Comment:
you're right, we can remove `core/balance` since the balance is a number in the chrysalis.
we should remove it in another PR, not in #48.